src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,52 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		info, err := os.Stat(path)
+		if os.IsNotExist(err) {
+			log.Fatalf("Script not found: %s", path)
+			os.Exit(1)
+		}
+		// get the mode bits and check that the script is executable by someone
+		mode := info.Mode()
+		if mode & 0111 == 0 {
+			log.Fatalf("Script not executable: %s", path)
+			os.Exit(1)
+		}
+		log.Infof("Running script: %s", path)
+		if ctx.DryRun == true {
+			log.Infof("- OK (dry) %s", path)
+			break
+		}
+		// pass kube context and the "global" config as a yaml environment variable
+		cmd := exec.Command(path)
+		if ctx.AnkhConfig.CurrentContext.Global != nil {
+			global, err := yaml.Marshal(ctx.AnkhConfig.CurrentContext.Global)
+			check(err)
+			cmd.Env = append(
+				os.Environ(),
+				"ANKH_CONFIG_GLOBAL=" + string(global),
+				"ANKH_KUBE_CONTEXT=" + string(ctx.AnkhConfig.CurrentContext.KubeContext))
+		}
+		var stdOut, stdErr bytes.Buffer
+		cmd.Stdout = &stdOut
+		cmd.Stderr = &stdErr
+		err = cmd.Run()
+		if err != nil {
+			log.Fatalf("- FAILED %s:\nstdout: %s\nstderr: %s", path, stdOut.String(), stdErr.String())
```

Comment:
we'll want `err` as a string here too. It's okay to deviate a little from the python implementation's logging style and experience.